.drone.yml

```
@@ -1,35 +1,102 @@
-pipeline:
-  test_api:
-    image: docker/compose:alpine-1.25.5
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-      - DOCKER_IP=172.17.0.1
-    commands:
-      - apk add bash curl
-      - ./bin/test.sh
-    when:
-      event: [push]
-  tagged_images_to_quay:
-    image: docker:17.09.1
-    secrets:
-      - docker_password
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-      - DOCKER_USERNAME=ukhomeofficedigital+acp_clamav
-    commands:
-      - bin/build.sh $DRONE_TAG
-    when:
-      event: [tag]
-  latest_images_to_quay:
-    image: docker:17.09.1
-    secrets:
-      - docker_password
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-      - DOCKER_USERNAME=ukhomeofficedigital+acp_clamav
-    commands:
-      - bin/build.sh latest
-    when:
-      event: push
-      branch: master
+---
+kind: pipeline
+type: kubernetes
+name: default
 
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: test_api
+  pull: if-not-exists
+  image: docker/compose:alpine-1.25.5
+  environment:
+    DOCKER_HOST: tcp://127.0.0.1:2375
+  commands:
+  - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
```

```suggestion
  - n=0; while [ "$n" -lt 60 ] && [ ! "$(docker stats --no-stream)" ]; do n=$(( n + 1 )); sleep 1; done
```
To fix the `sh: stats: unknown operand` error in the Drone build